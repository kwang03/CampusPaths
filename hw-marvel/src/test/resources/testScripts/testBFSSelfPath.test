#Tests the BFS when asked to find a self path

LoadGraph myGraph twoCharsMultipleConnections.csv

FindPath myGraph Char1 Char1

FindPath myGraph Char2 Char2