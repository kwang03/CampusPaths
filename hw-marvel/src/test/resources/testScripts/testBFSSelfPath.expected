#Tests the BFS when asked to find a self path

loaded graph myGraph

path from Char1 to Char1:

path from Char2 to Char2: